test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/ObjectExplorer/ObjectExplorerServiceTests.cs

```diff
@@ -26,12 +26,26 @@ public class ObjectExplorerServiceTests
         public async void CreateSessionAndExpandOnTheServerShouldReturnTheDatabases()
         {
             var query = "";
-            string uri = "CreateSessionAndExpand";
+            string uri = "CreateSessionAndExpandServer";
+            string databaseName = null;
+            using (SqlTestDb testDb = SqlTestDb.CreateNew(TestServerType.OnPrem, false, databaseName, query, uri))
+            {
+                var session = await CreateSession(null, uri);
+                await CreateSessionAndServerNode(testDb.DatabaseName, session);
+                CancelConnection(uri);
+            }
+        }
+
+        [Fact]
+        public async void CreateSessionAndExpandOnTheDatabaseShouldReturnDatabaseObjects()
```

Comment:
none of these tests pass master or empty string as database name. can you please add a test to verify that. the code coverage is dropped in this pr so it might be a bug or missing tests